docs/recurring.md

```diff
@@ -0,0 +1,183 @@
+
+```js
+// STEP #6a: Add a button handler
+document.getElementById('submit').addEventListener('click', () => {
+  Airwallex.createPaymentConsent({
+    intent_id: 'replace-with-your-intent-id', // intent id(Optional)
+    customer_id: 'replace-with-your-customer-id', // customer id
+    client_secret: 'replace-with-your-client-secret', // client secret
+    currency: 'replace-with-your-currency', // currency
+    element: cardElement // either the card element or card number element depends on the element you integrate,
+    next_triggered_by: 'customer' // 'merchant' for MIT and 'customer' for CIT
+  }).then((response) => {
+    // STEP #6b: Listen to the request response
+    /* Handle response */
+    window.alert(JSON.stringify(response));
+  });
+});
+```
+
+`Airwallex.createPaymentConsent` will take the cardNumber element or card element you mounted and create payment consent.
+
+More details about the `createPaymentConsent` function can be found [here](/docs#createPaymentConsent).
+
+### 3. Follow the split card element / card element integration document step 7-9
+
+[Split card element integration guide](https://github.com/airwallex/airwallex-payment-demo/blob/master/docs/splitcard.md#guide)
+
+[Card element integration guide](https://github.com/airwallex/airwallex-payment-demo/blob/master/docs/card.md#guide)
+
+
+## Guide for `Confirm payment using consent` with MIT flow
+For MIT consent, merchant could trigger the payment without shopper, so merchant could use the `confirmPaymentIntent` function to complete the transaction. [confirmPaymentIntent detail](https://github.com/airwallex/airwallex-payment-demo/tree/master/docs#confirmpaymentintent)
+
+## Guide for `Confirm payment using consent` with CIT flow
+For CIT consent, shopper will trigger a transaction and enter cvc number to finish the payment. So merchant need to integrate with cvc element.
+
+### 1. At the start of your file, import `airwallex-payment-elements`.
+
+```js
+import Airwallex from 'airwallex-payment-elements';
+```
+
+or add the bundle as a script in your HTML head
+
+```html
+<script src="https://checkout.airwallex.com/assets/bundle.x.x.x.min.js"></script>
+```
+
+Be sure to replace the x.x.x with the `airwallex-payment-elements` package version you'd like to use.
+
+### 2. Initialize the Airwallex package with the appropriate environment
+
+```js
+Airwallex.loadAirwallex({
+  env: 'demo', // Setup which Airwallex env('staging' | 'demo' | 'prod') to integrate with
+  origin: window.location.origin, // Setup your event target to receive the browser events message
+});
+```
+
+`loadAirwallex` takes in options to set up the payment environment. See docs for further customizations [here](/docs#loadAirwallex).
+
+The Airwallex package only needs to be mounted once in an application (and everytime the application reloads).
+
+### 3. Add empty containers for cvc element to be injected into and a submit button to trigger the payment request
+
+```html
```

Comment:
should we include an example list of payment consents?
```suggestion
```html
<div>List of payment consents</div>
```